app/schema.graphql

```diff
@@ -580,6 +581,32 @@ type Experiment implements Node {
   """Sequence number (1-based) of experiments belonging to the same dataset"""
   sequenceNumber: Int!
   runs(first: Int = 50, last: Int, after: String, before: String): ExperimentRunConnection!
+  annotationSummaries(first: Int = 50, last: Int, after: String, before: String): ExperimentAnnotationSummaryConnection!
```

Comment:
probably don't want this to be a connection since we probably will never paginate?

Reply:
addressed in 27266906302e399bebd3848f59a93b0f101f58aa